plugin_engine.py

```diff
@@ -96,8 +97,18 @@ def load_plugins(app_context: PluginAppContext):
             try:
                 plugin = importlib.import_module(name)
                 getattr(plugin, "__plugin_entrypoint__")(app_context)
+                active_plugins[name.split(".")[1]] = {
+                    "active": True,
+                    "author": getattr(plugin, "__author__"),
+                    "version": getattr(plugin, "__version__"),
+                    "description": getattr(plugin, "__doc__"),
```

Comment:
I would provide None as a default here.  Without default, we get AttributeError exception for non-existent module attribute. `__doc__` is part of module interface and will always exist (set to `None` w/o docstring), but we should provide default as well, just to be consistent.
```suggestion
                    "author": getattr(plugin, "__author__", None),
                    "version": getattr(plugin, "__version__", None),
                    "description": getattr(plugin, "__doc__", None),
```